PowerEditor/installer/sign-binaries.bat

```diff
@@ -0,0 +1,13 @@
+@ECHO OFF
+
+ECHO Start signing file: %1
+
+".\signCert\SignTool.exe" sign /f ".\signCert\npp_sign_cert.pfx" "%1"
+if errorlevel 1 goto SigningFailed
+goto SigningOK
+
+:SigningFailed
+echo Failed to sign file %1
+exit 1
+
+:SigningOK
```

Comment:
Please rename the file `sign-binaries.bat` to `sign-installers.bat`,
and use the following content instead:
 
```batch
@ECHO OFF

if [%SIGN%] == [] goto NoSignInstaller
if not %SIGN% == 1 goto NoSignInstaller

ECHO Start signing file: %1
%signBinary% "%1"

if errorlevel 1 goto SigningFailed
goto SigningOK

:SigningFailed
echo Failed to sign file %1
exit 1

:NoSignInstaller

:SigningOK
```

`%signBinary%` is already defined in `packageAll.bat`

Reply:
It is not that easy to get the `%signBinary%` from `packageAll.bat` -> `nsis compiler` -> then `sign-installers.bat`.

However, I have made some changes which should work (I could not test as I don't have cert and password handy). Also, I assume that you must have two env variable for `%NPP_CERT%` and `%NPP_CERT_PWD%`.